devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -137,4 +137,5 @@ def refresh() {
 def configure() {
 	log.debug "configure"
 	sendEvent(name: "checkInterval", value:20 * 60 + 2*60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	return zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 21600, 0x10)
```

Comment:
The commands sent here should be:
* read battery percentage
* send the enrollResponse to enroll the device in the IAS Zone
* configure battery reporting
* read the IAS Zone status